rails.rb

```diff
@@ -9,7 +9,20 @@
 # returns "Hello, World" for the given language, for at least 3 languages. It should default to returning English.
 #hello_world("es") should return "Hola Mundo"
 #hello_world("de") should return "Hallo Welt"
+lang = ["es", "de", "en"]
+def helloworld
 
+if lang === "es"
+    puts "Hola Mundo"
+elsif lang ==="de"
+    puts "Hallo Welt"
+else lang ==="en"
+    puts "Hello World"
+    
+greeting
+
+end 
+puts helloworld(lang ("es"))
 
```

Comment:
Couple things here - your if/else block is missing an end, you have an extra word inside the function, and instead of creating an array of language codes, just pass those in to your function calls as tests. You have all the right pieces here, just need to tweak it a little bit.

def helloworld lang
  if lang === "es"
      puts "Hola Mundo"
  elsif lang ==="de"
      puts "Hallo Welt"
  else
      puts "Hello World"
  end
end
helloworld "es"
helloworld "de"
helloworld "en"